### PR TITLE
Update footer

### PR DESCRIPTION
**Pull request by nishinji**
I made it look like this
スクリーンショット 2022-09-24 013811
I think it would be more useful to have links.

### CODE REVIEW

File: _includes/footer.html [hunk 1]
```diff
@@ -1,9 +1,56 @@
 {% raw %}
-<section class="text-white" id="copyright">
+<section class="text-white" id="footer">
     <div class="container">
         <div class="row">
+            <div class="col-lg-3 col-md-6 mb-4 mb-md-0">
+                <h3>　Main</h3>
```
Comment (EXtremeExploit) [inline _includes/footer.html:6]: extra space here

**Review by EXtremeExploit — CHANGES_REQUESTED** [review]

File: _includes/footer.html (hunk 1 above)
Comment (EXtremeExploit) [reply _includes/footer.html:6]: and in the other <h3> tags too

Comment (nishinji) [reply _includes/footer.html:6]: I know it's not good, but Main won't come right up the row.

Comment (EXtremeExploit) [reply _includes/footer.html:6]: ah i see now, no problem

**Review by EXtremeExploit — CHANGES_REQUESTED** [review]

File: _includes/footer.html [hunk 2]
```diff
@@ -1,9 +1,56 @@
...
+                    <li><a href="https://www.reddit.com/r/vita3k/">Reddit</a></li>
+                    <li><a href="https://twitter.com/vita3k">Twitter</a></li>
+                </ul>
+            </div>
+            <div class="col-lg-3 col-md-6 mb-4 mb-md-0">
+                <h3>　Repos</h3>
+                <ul>
+                    <li><a href="https://github.com/Vita3K">Vita3K</a></li>
+                    <li><a href="https://github.com/Vita3K/Vita3K.github.io">Website</a></li>
+                    <li><a href="https://github.com/Vita3K/compatibility">Compatibility</a></li>
```
[the remark this answers is not in the document]
Comment (nishinji) [reply _includes/footer.html:45]: fixed

**Comment by nishinji** [comment]
now
image

File: _includes/footer.html [hunk 3]
```diff
@@ -1,9 +1,56 @@
...
+                    <li><a href="https://www.reddit.com/r/vita3k/">Reddit</a></li>
+                    <li><a href="https://twitter.com/vita3k">Twitter</a></li>
+                </ul>
+            </div>
+            <div class="col-lg-3 col-md-6 mb-4 mb-md-0">
+                <h3>　Repos</h3>
+                <ul>
+                    <li><a href="https://github.com/Vita3K">Vita3K</a></li>
+                    <li><a href="https://github.com/Vita3K/compatibility">Compatibility</a></li>
+                    <li><a href="https://github.com/Vita3K/homebrew-compatibility">homebrew-compatibility</a></li>
```
Comment (EXtremeExploit) [inline _includes/footer.html:46]: homebrew-compatibility -> Homebrew Compatibility

**Pull request merged**